internal/manifest/os.go

```diff
@@ -74,8 +79,9 @@ type OSCustomizations struct {
 	// Do not install documentation
 	ExcludeDocs bool
 
-	Groups []users.Group
-	Users  []users.User
+	Groups   []users.Group
+	Users    []users.User
+	YUMRepos []rpmmd.RepoConfig
```

Comment:
Same here. Going to change this to a map to accept filename

Reply:
Alternatively you can define it as a new type in `rpmmd`, which would be IMO also reasonable. Something like:
```go
type YUMRepoFile map[string]RepoConfig
```